src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -336,22 +336,23 @@ def create_pool(self):
         """
         pool = self.get_pool(connect=False)
 
+        # Continue using the uuid until there are tests for both uuid and label
+        pool.use_label = False
+
         # Save the pool and uuid
         self.pool.append(pool)
         self.uuids.append(str(pool.uuid))
 
         return pool
 
-    def create_cont(self, pool, use_dfuse_uns=False,
-                    dfuse_uns_pool=None, dfuse_uns_cont=None,
+    def create_cont(self, pool, use_dfuse_uns=False, dfuse_uns_pool=None, dfuse_uns_cont=None,
```

Comment:
(pylint-arguments-renamed) Parameter 'chunk_size' has been renamed to 'pool' in overridden 'DataMoverTestBase.create_cont' method
(pylint-arguments-renamed) Parameter 'properties' has been renamed to 'use_dfuse_uns' in overridden 'DataMoverTestBase.create_cont' method

Reply:
I think we can ignore this one, as it exists in master as well and requires a more comprehensive fix